src/helpers/mod.rs

```diff
@@ -30,8 +32,10 @@ pub fn get_platform_name() -> &'static str {
 pub fn get_platform_name_download() -> &'static str {
     if cfg!(target_os = "windows") {
         "nvim-win64"
-    } else if cfg!(target_os = "macos") {
-        "nvim-macos"
+    } else if cfg!(target_os = "macos") && cfg!(target_arch = "arm64") {
```

Comment:
```suggestion
    } else if cfg!(target_os = "macos") && cfg!(target_arch = "aarch64") {
```

I also had to change this line for it to work for me locally. cc @gibfahn